### DISCUSSION TITLE
Can I customize toolbar controls? /minimize, maximize, close/

### DISCUSSION THREAD

**Discussion by tgldr** [microsoft/react-native-windows#12440] [General]
Hi, Can I change minimize, maximize, close buttons size?

<img width="940" alt="titlebar-overview" src="https://github.com/microsoft/react-native-windows/assets/8258715/78d567cb-5329-4038-a6de-00b85fc64a01">

**Comment by ghost** [comment]
According to the documentation, even with the [full customization](https://learn.microsoft.com/en-us/windows/apps/develop/title-bar?tabs=winui2#full-customization), it is still the system's responsibility to control these buttons😕. However, there is an example in the playground with Win32 available here: [Win32 playground example](https://github.com/microsoft/react-native-windows/blob/main/packages/playground/README.md#playground-win32)😃, where Win32 allows for a much more solid customization. Am I correct, @chrisglein?

**Reply by chrisglein** [reply]
That's my understanding. In order to get full titlebar customization you need to do it in Win32. Some apps will be Win32 apps and have a XAML island within that just so they can customize the titlebar canvas but then have WinUI content.
